js/content/store.js

```diff
@@ -2662,53 +2660,41 @@ let FundsPageClass = (function(){
             );
         }
 
-        let currency = await Price.parseFromString(price, Currency.storeCurrency);
+        let currency = CurrencyRegistry.fromType(Currency.storeCurrency);
+        let minValue = currency.valueOf(price);
+        let symbol = currency.format.symbol;
+        let step = currency.format.decimalPlaces === 0 ? 1 : .01;
 
-        let inputel = newel.querySelector((giftcard ? "#es_custom_money_amount_wrapper" : ".price"));
-        HTML.inner(inputel, "<input type='number' id='es_custom_money_amount' class='es_text_input money' min='" + currency.value + "' step='.01' value='" + currency.value +"'>");
-        // TODO currency symbol
+        HTML.inner(
+            newel.querySelector(giftcard ? "#es_custom_money_amount_wrapper" : ".price"),
+            `${symbol}<input type="number" id="es_custom_money_amount" class="es_text_input money" min="${minValue}" step="${step}" value="${minValue}">`
```

Comment:
You can read from the currency config whether it's to the left or right of the number and then insert it properly

Reply:
uh I totally forgot about this